runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java

```diff
@@ -93,6 +94,15 @@ public class EmoteClue extends ClueScroll implements LocationClueScroll
 		ItemVariationMapping.getVariations(PHARAOHS_SCEPTRE).stream()
 			.map(ItemRequirements::item)
 			.toArray(SingleItemRequirement[]::new));
+	private static final AnyRequirementCollection ACTIVE_CRYSTAL_BOW_OR_ACTIVE_BOW_OF_FAERDHINEN = any("Crystal Bow or Bow of Faerdhinen",
```

Comment:
Slightly less wordy var name. Otherwise lgtm
```suggestion
	private static final AnyRequirementCollection ACTIVE_CRYSTAL_BOW_OR_BOW_OF_FAERDHINEN = any("Crystal Bow or Bow of Faerdhinen",
```

Reply:
Update committed and rebased into initial commit.